### ISSUE TITLE
XML prologue is accepted by `validate` but not by `Xeno.DOM.parse`

### ISSUE THREAD

**Issue by ocramz** [ocramz/xeno#16]
```
λ> isRight $ Text.XML.Hexml.parse "<?xml version=\"1.1\"?>\n<greeting>Hello, world!</greeting>"
True
λ> isRight $ Xeno.DOM.parse "<?xml version=\"1.1\"?>\n<greeting>Hello, world!</greeting>"
False
```

**Comment by qrilka** [comment]
@ocramz this one was fixed in #11 which also included this specific test case

**Comment by ocramz** [comment]
Great @qrilka , thanks ^^
